protobuf/node.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
```

Comment:
Split node defininiton into its own proto file . This way for node update there won't be any change to the SDK